configure.ac

```diff
@@ -12,7 +12,7 @@ AC_INIT([flatpak-builder],
         [flatpak-builder],
         [http://flatpak.org/])
 
-GLIB_REQS=2.44
+GLIB_REQS=2.66
```

Comment:
This is a massive bump. I don't think its acceptable.

Reply:
What older OSs are you interested in running flatpak-builder on? When there's a GLib feature that we can benefit from as much as the existence of GUri, I think the required version should be driven by which OSs we want to support, rather than by backwards-compat for the sake of backwards-compat.

Reply:
It's flatpak-builder, not flatpak. Do we really need to support _building_ flatpaks on something older than Debian Stable? There is `org.flatpak.Builder` flatpak after all.

Reply:
I didn't track down who has it. Debian Stable having it is a reasonable metric for this being OK.